R/render.r

```diff
@@ -162,6 +170,32 @@ check_open_graph <- function(og) {
   og[intersect(supported_fields, names(og))]
 }
 
+get_bs_version <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  template <- pkg$meta[["template"]]
+
+  if (is.null(template$bootstrap)) {
+    return(3)
+  }
+
+  if (!template$bootstrap %in% c(3, 4)) {
```

Comment:
This would be another good place for an early return:

```R
if (template$bootstrap %in% c(3, 4)) {
  return(template$bootstrap)
}
```

I like this style of function construction because it's like tackling a problem starting from the easiest parts.